azure-pipelines.yml

```diff
@@ -107,7 +107,7 @@ stages:
           submodules: false
           coverage: codecov
           toxdeps: tox-pypi-filter
-          posargs: -n auto --dist loadgroup
+          posargs: -n auto --dist loadscope
```

Comment:
not sure if this is the correct one.

Reply:
Use `loadfile`, it's what I did in GH actions

Reply:
This is updating the cron stage although I think only oldestdeps should need it? Could just update SecondPhaseTests with
```yaml
- linux: py38-oldestdeps
  posargs: -n auto --dist loadfile
```

Reply:
Woops, clearly was paying no attention to what block I was editing.